ansible/group_vars/all/vars

```diff
@@ -0,0 +1,23 @@
+# Default test env templates
+testenv_testbed_vmware_template: Template-testbed-201803050718
+testenv_controller_vmware_template: Template-CentOS-7.4
```

Comment:
Do we need these defaults here?

Reply:
I was thinking about these as a substitute to Jenkins parameters/contents. I think they can stay for now and in the future we'll move jenkins variables to vars/vault.